bundles/ui/org.eclipse.smarthome.ui.login/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Again, please let us try to keep the number of additional bundles small: This one does not have any too specific dependencies and consists out of a single class only - this should imho be moved to the smarthome.ui bundle.